docs/topics/kmm/kmm-getting-started.md

```diff
@@ -1,22 +1,55 @@
-[//]: # (title: Getting started)
+[//]: # (title: Get started with Kotlin Multiplatform Mobile)
+[//]: # (description: Simplify cross-platform app development with Kotlin Multiplatform Mobile. Create a single codebase
+for the business logic of your iOS and Android apps.)
 
-## Get familiar with KMM
+> Kotlin Multiplatform Mobile is in [Alpha](components-stability.md). Language features and tooling may change in future
+> Kotlin versions.
+> 
+> Beta is expected in spring 2022. See [KMM Beta Roadmap](https://youtu.be/LE-Dy9XCLxk) to learn about upcoming features.
```

Comment:
We have renamed the video. I think that it's better to link to the [corresponding blog post](https://blog.jetbrains.com/kotlin/2021/10/kmm-beta-roadmap-video-highlights/) (will be renamed as well) that describes highlights and embeds this video. Please consult with @KaterinaPetrova .